packages/code-snippet-block/src/settings.ts

```diff
@@ -0,0 +1,326 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { DropdownSize, IconEnum, MultiInputLayout } from '@frontify/fondue';
+import { appendUnit, numericalOrPixelRule } from '@frontify/guideline-blocks-shared';
+import { BlockSettings, Bundle } from '@frontify/guideline-blocks-settings';
+import { BORDER_COLOR_DEFAULT_VALUE, DEFAULT_THEME_VALUE } from './constants';
+
+export const settings: BlockSettings = {
+    main: [
+        {
+            id: 'language',
+            type: 'dropdown',
+            defaultValue: 'html',
+            size: DropdownSize.Large,
+            choices: [
+                {
+                    value: 'html',
+                    icon: IconEnum.Snippet,
+                    label: 'HTML',
+                },
+                {
+                    value: 'css',
+                    icon: IconEnum.Snippet,
+                    label: 'CSS',
+                },
+                {
+                    value: 'js',
+                    icon: IconEnum.Snippet,
+                    label: 'JS',
+                },
+                {
+                    value: 'jsx',
+                    icon: IconEnum.Snippet,
+                    label: 'JSX',
+                },
+                {
+                    value: 'ts',
+                    icon: IconEnum.Snippet,
+                    label: 'TS',
+                },
+                {
+                    value: 'json',
+                    icon: IconEnum.Snippet,
+                    label: 'JSON',
+                },
+                {
+                    value: 'php',
+                    icon: IconEnum.Snippet,
+                    label: 'PHP',
+                },
+            ],
+        },
+    ],
+    basics: [],
```

Comment:
I somehow have a discussion in mind that `basics` settings always has at least one value in it. But I'm not sure... Have you discussed the settings with a designer?

Reply:
I have not. But I don't think anything changed past eight months. Otherwise, the Designer would probably notify me.